src/test/java/nl/atos/ontwikkelstraat/unit/controller/FormControllerTest.java

```diff
@@ -61,4 +61,9 @@ public void addNewPostWrongEndpoint() throws Exception {
                 .andExpect(status().isOk());
     }
 
+    @Test
+    public void setNAW() throws Exception {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://www.codacy.com/app/vinsterdamus/Ontwikkelstraat/file/2327518792/issues/source?bid=3138095&fileBranchId=3206756#l65)